Kickstarter.xcodeproj/project.pbxproj

```diff
@@ -2842,11 +2837,11 @@
 				D6B4F0012107B3D00079159D /* SettingsNewslettersViewControllerTests.swift */,
 				D6AD9A7F20EA720F0015A18B /* SettingsNotificationsViewController.swift */,
 				D6B6F91720EE955B00A295F7 /* SettingsNotificationsViewControllerTests.swift */,
-				771E3C622289DBA8003E7CF1 /* SheetOverlayViewController.swift */,
 				D703FC6820F7E2EC004A272D /* SettingsPrivacyViewController.swift */,
 				D72370932119139D001EA4CA /* SettingsPrivacyViewControllerTests.swift */,
 				77216D9720F3D2E40061BE82 /* SettingsViewController.swift */,
 				D6E7DAFB22089F9800689BD6 /* SettingsViewControllerTests.swift */,
+				771E3C622289DBA8003E7CF1 /* SheetOverlayViewController.swift */,
```

Comment:
Applied sorting of files alphabetically after renaming so that's probably the reason here